dangerfile.ts

```diff
@@ -109,11 +109,16 @@ const checkPrDescription: () => void = () => {
 
 const checkCodeChanges: () => void = () => {
   // Request changes to package source code to also include changes to tests.
-  const hasCodeChanges = allFiles.some((p) => !!p.match(/^src\/.*\.[jt]sx?/))
+  const hasSourceCodeChanges = allFiles.some(
+    (p) => !!p.match(/^src\/.((?!\.(stories|test)\.).)*\.[jt]sx?/)
+  )
```

Comment:
Changes to Dangerfile to avoid this message when only storybook is updated:

<img width="798" height="122" alt="image" src="https://github.com/user-attachments/assets/ea86cd07-7306-4ef9-9c89-820885db5874" />

<img width="494" height="235" alt="image" src="https://github.com/user-attachments/assets/4a1191a2-018a-4752-a9bf-4a0e52a8d32b" />